packages/Telephony/Readme.md

```diff
@@ -105,12 +105,43 @@ The Stop Recording action stops recording of the conversation. Note that it is n
 #### Failures
 * When a stop recording result is received and indicates error, the dialog throws an `ErrorResponseException`.
 
-## **DTMF Batching**
-_In progress_
+## **Aggregate DTMF Input (n)**
```

Comment:
The current design allows to aggregate more than speech. Consider describing how it the recommended usage for non-DTMF.

Reply:
How do you feel about undocumented features :D